vendor/licenses/git_submodule/vscode-liquidsoap.dep.yml

```
@@ -0,0 +1,33 @@
+---
+name: vscode-liquidsoap
+version: 037bccaf1f74825e76ab6bd0ffc27eaac902ac78
+type: git_submodule
+homepage: https://github.com/savonet/vscode-liquidsoap.git
+license: other
```

Looks like `licensed` couldn't successfully determine the license, possibly because of the first line, so we need to manually change this.

```suggestion
license: mit
```